rhizome/host/lib/vm_setup.rb

```diff
@@ -599,6 +600,12 @@ def write_user_data(unix_user, public_keys, swap_size_bytes, boot_image)
   - [nft, add, rule, ip6, filter, output, ip6, daddr, 'fd00:0b1c:100d:5AFE::/64', meta, skuid, "!=", 0, tcp, flags, syn, reject, with, tcp, reset]
 NFT_ADD_COMMS
 
+    init_script_cmd = if init_script
+      <<INIT_SCRIPT_CMD
+  - [bash, -c, #{yq(init_script)}, #{init_script_args.to_s.shellsplit.map { |arg| yq(arg) }.unshift("bash").join(", ")}]
```

Comment:
Can this be simplified and brought into line with however cloud-init implements shell execution when passed a string as an array element?

https://cloudinit.readthedocs.io/en/latest/reference/examples.html, see `ls -l /root`

Reply:
If we didn't support init script args, I think the simplified version (`  - #{yq(init_script)}`) would work. However, I'm not sure how you would support init script args with that approach. Do you have any ideas?